### ISSUE TITLE
Add number shortcut labels to quick preset buttons

### ISSUE THREAD

**Issue by quincylvania** [openstreetmap/iD#6118]
Where there are lots of favorites and recents, it's hard to determine which keyboard shortcut to use at a glance. We should add small labels with the numbers of the keys.

<img width="783" alt="Screen Shot 2019-03-27 at 10 08 19 AM" src="https://user-images.githubusercontent.com/2046746/55082583-54c4ec00-5078-11e9-8d6d-2ad10323e8ec.png">

**Comment by BjornRasmussen** [comment]
I think that people will figure the keyboard shortcuts or pretty quickly by hovering - you can't click on one of the presets without first hovering over it. 
Adding the numbers would clutter the screen with even more information to absorb, making iD more confusing for new mappers. Also, the numbers would not be useful to mobile users.

**Comment by BjornRasmussen** [comment]
Adding to my message above, I think that keeping the UI as simple as practically possible is a good idea. Anything that isn't really necessary should be kept to allow users to focus on more important information on the screen.

**Comment by quincylvania** [comment]
@BjornRasmussen Thanks for your thoughts.

> I think that keeping the UI as simple as practically possible is a good idea.

Agreed! We try to keep iD as simple as we can. We also try to balance the needs of power users.

The idea behind adding number labels is that needing to hover in order to see the shortcut negates the usefulness of a shortcut. Normally you can just memorize a shortcut and it'll be fine, but in this case the favorite and recent presets could constantly be changing.

We would probably only show the numbers after a minimum number of preset buttons are on screen. We could hide them at narrow screen sizes. And if we can't find a way for it to look good, then we won't do it.

**Comment by bhousel** [comment]
Yeah we definitely need something like this if buttons come and go and their positions are not stable.

I'd be ok with switching the hotkeys to <kbd>⌘1</kbd><kbd>⌘2</kbd><kbd>⌘3</kbd>, etc.. and on keydown of the <kbd>⌘</kbd>, change the below-the-button hint text to show the shortcut texts.

That's nice because it stays out of the way of mobile users who will never use the key shortcuts anyway and doesn't affect anything on the buttons, which are already cramped.

**Comment by todrobbins** [comment]
@bhousel I like that idea, but won't `⌘1``⌘2``⌘3` conflict with browser's assignment of those hotkeys to tabs 1, 2, 3, etc.?

**Comment by k-yle** [comment]
I'm going to close this issue, since it's an issue in the v3 prototype version, which is (sadly) abandoned and [no longer deployed](https://github.com/openstreetmap/iD/commit/36f4eeb7e24ba3ffd16f0d48780f3c56d39f2bd1).
